packages/contracts/contracts/YieldGate.sol

```diff
@@ -68,13 +70,27 @@ contract YieldGate {
 }
 
 contract BeneficiaryPool {
```

Comment:
Wanted to ask this for quite some time. Is it good practice to keep both contract definitions in the same file?

Reply:
I just did it in the context of the hackathon and then never moved it to a separate file... I'd say it's not bad practice if multiple contracts are closely related to keep them in the same file. But as complexity of individual contracts grows, they should probably be put into separate files. I can do it in this PR.

Reply:
Ok, thanks for clarification. You decide whether to move or not :)